css/main.css

```diff
@@ -90,7 +90,7 @@ lines .wrap {
 }
 
 .main-header-div {
-  padding: 1rem;
+  padding: 10rem;
```

Comment:
10rem in this case would be 10x16px
160 px for text would be very large

Reply:
Does it matter whether the element is text or not.....?